data/beaconrestapi/src/test/java/tech/pegasys/teku/beaconrestapi/handlers/v1/beacon/GetGenesisTest.java

```diff
@@ -51,6 +51,6 @@ public void shouldReturnGenesisInformation() throws Exception {
     when(chainDataProvider.getStateGenesisData()).thenReturn(expectedGenesisData);
```

Comment:
not sure this when will even get called in this test now

Reply:
Hopefully we have some kind of integration test that gives us actual data, this test is borderline able to be deleted, its not doing anything any more